.strict-typing

```diff
@@ -155,6 +155,7 @@ homeassistant.components.image_processing.*
 homeassistant.components.input_button.*
 homeassistant.components.input_select.*
 homeassistant.components.integration.*
+homeassistant.components.isy994.*
```

Comment:
I think this comes from an incorrect rebase.

Reply:
Reverted.